### ISSUE TITLE
Review Instances are seeing fles missing on http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/ 

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#11288]
Initial report at https://dsva.slack.com/archives/CDHBKAL9W/p1666218324226389 by @alexandec on Oct 29 at 3:29pm PT. 

```
We are seeing review instances fail to build, and they are failing on five particular images. I was able to reproduce the download failures locally with yarn build --verbose as well. The images are:
http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/1_1_square_medium_thumbnail/public/2022-10/Maiah%202022%20headshot%20%28Edited%29.JPG 
http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/1_1_square_medium_thumbnail/public/2022-10/Ahmaya%20Mustafa%20PGY1%20%5BIndividual%5D.jpg
http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/1_1_square_medium_thumbnail/public/2022-10/Orelon%20edited.JPG
http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/1_1_square_medium_thumbnail/public/2022-10/LaNeria%20edited.JPG
http://cms-content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_1_medium_thumbnail/public/2022-10/LaNeria%20edited.JPG
I'm able to load other images from that server, but not these 5. Any idea why? ([thread for reference](https://dsva.slack.com/archives/CBU0KDSB1/p1666104622300489))
```

**Comment by ElijahLynn** [comment]
Issue was that the previews hadn't been built since October 3rd, I rebuilt them manually at 8:43pm PT and it was confirmed by @it-harrison at 7:08am PT this morning. 

![image](https://user-images.githubusercontent.com/1504756/197011129-e30c267e-5e5e-466d-808d-51a7519aa752.png)

**Comment by ElijahLynn** [comment]
Also, the root cause of this seems to be some Tugboat upgrades I did on Oct. 3rd > https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6306#issuecomment-1264133294. 

The issue seems to be resolved now and the previews auto-refreshed at 4am PT per the schedule we have set and I confirmed they were refreshed at the same time the base refreshed. 
![image](https://user-images.githubusercontent.com/1504756/197012483-38a3ea4e-5fb6-4f4b-bcdf-4ba5c4721ac3.png)
